src/main/java/randoop/reflection/OperationModel.java

```diff
@@ -162,6 +166,8 @@ public static OperationModel createModel(
 
     OmitMethodsPredicate omitPredicate = new OmitMethodsPredicate(omitMethods);
 
+    model.omitMethodsPredicate = omitPredicate;
+
     model.addOperationsFromClasses(
         model.classTypes, visibility, reflectionPredicate, omitPredicate, operationSpecifications);
```

Comment:
The model now contains a field with this object in it.  So, does this need to be passed in any more?